specification/hdinsight/resource-manager/readme.md

```diff
@@ -71,3 +70,18 @@ Please also specify `--go-sdk-folder=<path to the root directory of your azure-s
 ``` yaml $(tag) == 'package-2015-03-preview' && $(go)
 output-folder: $(go-sdk-folder)/services/hdinsight/mgmt/2015-03-01-preview/hdinsight
 ```
+
+
+## C# 
+
+These settings apply only when `--csharp` is specified on the command line.
+Please also specify `--csharp-sdks-folder=<path to "SDKs" directory of your azure-sdk-for-net clone>`.
+
+``` yaml $(csharp)
+csharp:
+  azure-arm: true
+  license-header: MICROSOFT_APACHE_NO_VERSION

```

Comment:
CELA recommands MIT, and it's usually what is used here. Random example:
https://github.com/Azure/azure-rest-api-specs/tree/current/specification/network/resource-manager#c

Could you confirm this is an actual choice, and not random?